qa/rpc-tests/cfund-paymentrequest-state-accept-expired-proposal.py

```diff
@@ -150,7 +151,7 @@ def run_test(self):
         self.nodes[0].paymentrequestvote(paymentrequestid0, "remove")
 
         assert(self.nodes[0].getpaymentrequest(paymentrequestid0)["state"] == 0)
-        assert(self.nodes[0].getpaymentrequest(paymentrequestid0)["status"] == "accepted waiting for end of voting period")
+        assert_equal(self.nodes[0].getpaymentrequest(paymentrequestid0)["status"], "accepted waiting for end of voting period")
```

Comment:
any benefit changing from `assert` with the string comparison to `assert_equal`? or just for consistency?

Reply:
just for debug reasons